[PLAT-2774] Launch form query parameter support (#685)
seqeralabs/docs: platform-cloud/docs/launch/launchpad.md

```
%7B"key1":%20"value1",%20"key2":%20"value2"%7D
```

Platform will ignore added percent-encoding characters in form fields, so you do not need to remove them manually before submitting your pipeline launch. 
:::

### Supported URL query parameters and corresponding launch form fields

| **Launch form field**                          | **Query parameter name**  |
|------------------------------------------------|---------------------------|
| **General config**                             |                           |
| Pipeline to launch                             | `pipeline`                  |
| Revision number                                | `revision`                  |
| Config profiles                                | `configProfiles`            |
| Workflow run name                              | `runName`                   |
| Labels                                         | `labelIds`                  |
| Compute environment                            | `computeEnvId`              |
| Work directory                                 | `workDir`                   |
| **Run parameters**                             |                           |

> | Run parameters                                 | `paramsText`                |

Author: ```suggestion
| Pipeline-specific run parameters                                 | `paramsText`                |
```

:::note
Pipeline names must be unique per workspace.
:::

## Populate launch form with URL query parameters

The launch form can populate fields with values passed as URL query parameters. For example, append `?revision=master` to your launch URL to prefill the **Revision** field with `master`. This feature is useful for Platform administrators to provide custom pipeline launch URLs to users in order to hard-code required run and pipeline parameters for every run.

Platform validates run parameters passed via the launch URL in the following way:

> - The validity of parameter names is **not** validated. You must provide valid and supported parameters for launch form fields to be populated without error. See supported parameter names in the following section. 

Reviewer: ```suggestion
- Parameter names are **not** validated. You must provide valid and supported parameters for launch form fields to be populated without error. See supported parameter names in the following section.
```
